src/utils/model.js

```diff
@@ -14,17 +14,30 @@ async function fetchAll(urls) {
   }
 }
 
+export const ModelIds = {
+  baseline: 0,
+  strictDistancingNow: 1,
+  weakDistancingNow: 7,
+  containNow: 2,
+};
+
 export function useModelDatas(location) {
   const [modelDatas, setModelDatas] = useState(null);
-
   useEffect(() => {
     async function fetchData() {
-      let urls = Array.from(
-        { length: 8 },
-        (x, i) => `/data/${location}.${i}.json`,
-      );
+      let urls = [
+        ModelIds.baseline,
+        ModelIds.strictDistancingNow,
+        ModelIds.weakDistancingNow,
+        ModelIds.containNow,
+      ].map(i => `/data/${location}.${i}.json`);
       let loadedModelDatas = await fetchAll(urls);
-      setModelDatas(loadedModelDatas);
+      setModelDatas({
+        baseline: loadedModelDatas[0],
```

Comment:
should the indexes also use ModelIds.baseline instead of just [0]

Reply:
no - these are serially sequential as returned from fetchAll

Reply:
ahh yes makes sense